packages/front-end/components/HomePage/ExperimentsGetStarted.tsx

```diff
@@ -119,12 +119,12 @@ const ExperimentsGetStarted = ({
               setDataSourceOpen(false);
               setDataSourceQueriesOpen(true);
             }}
-            importSampleData={
-              !hasDataSource &&
-              allowImport &&
-              !hasSampleExperiment &&
-              importSampleData("datasource-form")
-            }
+            // importSampleData={
+            //   !hasDataSource &&
+            //   allowImport &&
+            //   !hasSampleExperiment &&
+            //   importSampleData("datasource-form")
+            // }
```

Comment:
Do we want to add a sample data option to this new modal?

Reply:
Added